tests/functional/codegen/calling_convention/test_external_contract_calls.py

```diff
@@ -1151,7 +1171,8 @@ def foo(arg2: int128) -> invalid: view
 def bar(arg1: address, arg2: int128) -> int128:
     return Foo(arg1).foo(arg2)
 """
-    assert_tx_failed(lambda: get_contract(contract), exception=UnknownType)
+    with tx_failed(exception=UnknownType):
```

Comment:
similar - i think this can just be with `pytest.raises(UnknownType): compile_code(...)`